### DISCUSSION TITLE
May I know what is backdoor_server port used for?

### DISCUSSION THREAD

**Discussion by minkeetan** [sevdokimov/log-viewer#83] [Q&A]
Hi sevdokimov,

There is a setting log-viewer.backdoor_server.port=9595 in config.conf. 

May I know what it is used for? Can I mask it?

**Comment by sevdokimov** [comment]
This port is needed if you have several instances on Log viewer in a cluster. Log viewer can open logs located on different nodes, in this case, one log viewer loads parts of logs from other logviewer instances using `log-viewer.backdoor_server.port` port. This port should not be accessible outside the cluster.
